opennlp-tools/src/test/resources/opennlp/tools/formats/leipzig/samples/.hidden

```diff
@@ -0,0 +1 @@
+Nothing in here
```

Comment:
Do we need to add exceptions for these files to pass the build without rat errors?

Reply:
This seems to work